### ISSUE TITLE
Call for interest in teaching/helping! When and which lesson(s)?

### ISSUE THREAD

**Issue by naoe-tatara** [naoe-tatara/2001-autumn-neic-online#1]
If you are interested in teaching/helping in Self-Organized Workshop(s) in Nordics, please leave a comment to this issue with the following info:
- Lesson (with URL) / as instructor or helper - *You can suggest any lesson but be aware that if it is not core curriculum there might be fewer people who can be co-instructor or helper.*
- When you wish to teach/help (month, week nr)
- Any dates/time you cannot join or want to avoid

ex.
- Lesson and role:
  - The Unix Shell(http://swcarpentry.github.io/shell-novice/) as instructor 
  - Version Control with Git (http://swcarpentry.github.io/git-novice/) as instructor
  - Plotting and Programming in Python (http://swcarpentry.github.io/python-novice-gapminder/) as helper
  - Introduction to Conda for (Data) Scientists (https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/) as helper

- When: End of August to middle of September
- Cannot: Sep. 2nd 9:00-12:00 CEST
- Avoid: Wednesdays after 15:00 CEST

**Comment by annajiat** [comment]
Lesson and role:

- The Unix Shell (http://swcarpentry.github.io/shell-novice/) as instructor
- Version Control with Git (http://swcarpentry.github.io/git-novice/) as instructor
- Version Control with Git (https://carpentries-incubator.github.io/git-novice-branch-pr/) as instructor
- Plotting and Programming in Python (http://swcarpentry.github.io/python-novice-gapminder/) as instructor
- Programming with Python (https://swcarpentry.github.io/python-novice-inflammation/) as instructor
- An Introduction to Java Programming (https://carpentries-incubator.github.io/java-intro/) as instructor
- Programming with R (http://swcarpentry.github.io/r-novice-inflammation) as helper
- R for Reproducible Scientific Analysis (http://swcarpentry.github.io/r-novice-gapminder) as helper
- Data Organization in Spreadsheets for Humanities (https://carpentries-incubator.github.io/spreadsheet-humanities-lesson/) as instructor
- Introduction to Conda for (Data) Scientists (https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/) as helper
- Extra Unix Shell Material (https://carpentries-incubator.github.io/shell-extras/) as helper
- Introduction to High-Performance Computing (https://carpentries-incubator.github.io/hpc-intro/) as helper
- Would be open to helping in other lessons as well. Please discuss.

When: Middle of July to end of September

Cannot: Fridays/Saturdays 2pm to 6pm CEST (UTC 12pm to 4pm)

Avoid: 
- July 21-22
- August 2-13, 17-19
- September 6-17

**Comment by RaphaelaHeil** [comment]
- Lesson and role:

	- as instructor:
		- The Unix Shell (http://swcarpentry.github.io/shell-novice/)
		- Programming with Python (https://swcarpentry.github.io/python-novice-inflammation/)
		- Plotting and Programming in Python (http://swcarpentry.github.io/python-novice-gapminder/)
		- Library Carpentry: Introduction to Working with Data (Regular Expressions) (https://librarycarpentry.org/lc-data-intro/)
		- Library Carpentry: The UNIX Shell (https://librarycarpentry.org/lc-shell/)
		- Python Intro for Libraries (https://librarycarpentry.org/lc-python-intro/)
	- as helper: 
		- Version Control with Git (http://swcarpentry.github.io/git-novice/)
		- Library Carpentry: Introduction to Git (https://librarycarpentry.org/lc-git/)


- When: August, September

- Cannot: 05.-10. September

- Avoid: Monday afternoons

**Comment by lucaferranti** [comment]
- Lessons and roles
  - unix shell (http://swcarpentry.github.io/shell-novice/), instructor/helper
  - version control with git (http://swcarpentry.github.io/git-novice/ ), instructor/helper
  - programming with python (https://swcarpentry.github.io/python-novice-inflammation/ ), instructor/helper
  - plotting with python (http://swcarpentry.github.io/python-novice-gapminder/), instructor/helper
  - programming with matlab (http://swcarpentry.github.io/matlab-novice-inflammation/), instructor/helper
  - corresponding topics (unix, git, python) from data and library carpentries as instructor/helper
  - everything julia related as instructor/helper, there's a lesson in the incubator atm, but it's still pre-alpha.
  
- when: july- december
- cannot give specific dates of when I can or cannot, as my work schedule is pretty dynamic, the earlier something is agreed, the higher the chances I can fix it.

**Comment by ailisarre** [comment]
I am interested in helping out with Self-Organized Workshops. There are a few lessons I could contribute as a helper. In time I would like to contribute as an instructor, but in August and September I will have other teaching duties to prioritize. 

I can assist with these lessons as a helper: 
The Unix Shell (http://swcarpentry.github.io/shell-novice/) 
Version Control with Git (http://swcarpentry.github.io/git-novice/)  

Data organization in spreadsheets: 
Data Organization in Spreadsheets for Humanities (https://carpentries-incubator.github.io/spreadsheet-humanities-lesson/)
Data Organization in Spreadsheets for Ecologists (https://datacarpentry.org/spreadsheet-ecology-lesson/)

Data Cleaning with OpenRefine:
Data Cleaning with OpenRefine for Ecologists (https://datacarpentry.org/OpenRefine-ecology-lesson/)

Any introductory lesson to R, or Data visualization in R: 
Data Analysis and Visualization in R for Ecologists (https://datacarpentry.org/R-ecology-lesson/)
R for Social Scientists (https://datacarpentry.org/r-socialsci/)
http://swcarpentry.github.io/r-novice-gapminder/

When: Sept-Dec.
Cannot: week 33 and 34 (16.-27.aug)

**Comment by naoe-tatara** [comment]
Thank you all for the interest! (sorry for taking time to get back)

Regarding timing, it seems the week of August 30 - September 3 is the most promising.
From what are written here, I would suggest going for; either of, OR, a combination of, Shell or/and Python workshop. What do you think? We can discuss which lesson/episodes we would use later :)

FYI, CodeRefinery is most likely to run a 6-half-day workshop in Autumn. As the current project funding ends at the end of October, it is likely to be latest before the middle of October. In addition, there are other discussions about running different types of workshops on Singularity/Container, Julia, and Matlab+Fortran+Python interface, all somewhere August-September. 

In my opinion, except for the CodeRefinery workshop, Carpentries workshop and other above mentioned workshops could have different target groups, and in that sense probably nice to run one workshop with Shell+Python targeting sort of novice people with programming and get them ready for CodeRefinery workshop and further?

**Comment by annajiat** [comment]
@naoe-tatara 
yes, shell+python as a prep for CodeRefinery workshop sounds good.

**Comment by Nazeeefa** [comment]
Thanks Naoe! I'd like to contribute to following as helper if need be:

- The Unix Shell (http://swcarpentry.github.io/shell-novice/)
- Library Carpentry: The UNIX Shell (https://librarycarpentry.org/lc-shell/)
- Data visualization in R
- Anything related to data visualisation that could also be a learning opportunity for me as a helper :D

**Comment by naoe-tatara** [comment]
Hi All, 

Might be repetitive for some of you, but FYI; the background of the initiation of Self-Organized Workshop (SOW) by CodeRefinery/NeIC is to make a place for mostly newly certified instructors to have experience in teaching with a bit easier (low-threshold) condition by enabling single-lesson based workshop. 

Given that 1) Annajiat has a lot of experiences in teaching at COWs, 2) Both Raphaela and Luca will most likely teach at the SWC-Centrally-Organized Workshop at Univ. of Bergen somewhere in Sep-Oct, 3) Aili does not have a chance to teach in the week that was suggested above, 4) there are some people who are going to take instructor training soon or in the process of checking-out, and 5) planning of a CodeRefinery workshop in Autumn by the current project is at the moment less prioritized as it is heading towards the end than the works towards the start of a new phase, I would like to postpone the planning of this SOW to later time (e.g. end of Oct) than the last week of Aug - first week of Sep. Hope it works for all.
